src/App.js

```diff
@@ -153,56 +177,30 @@ class App extends Component {
 
   handlePayment = async (transactionData, paymentParams, recipient) => {
     const res = await this.state.mainframe.payments.payContact(paymentParams);
+    let hash
```

Comment:
unsure why this var is needed

Reply:
we are using it below to save to Firebase..
Since I already have it, I don't need to use the one from state... and state is async.. so we are not entirely sure it will be there, although it should be fine...

Reply:
ah i missed it being used below! makes sense